nexus_constructor/stream_fields_widget.py

```diff
@@ -416,17 +425,19 @@ def fill_in_existing_f142_fields(self, field: h5py.Group):
                 self.f142_nexus_to_spinner_ui_element.items(), field
             )
 
-    def update_existing_stream_info(self, field: h5py.Group):
+    def update_existing_stream_info(self, field):
         """
         Fill in stream fields and properties into the new UI field.
         :param field: The stream group
         :param new_ui_field: The new UI field to be filled in
         """
-        raise NotImplementedError
-        schema = field["writer_module"][()]
-        self.schema_combo.setCurrentText(str(schema))
-        self.topic_line_edit.setText(str(field["topic"][()]))
-        self.source_line_edit.setText(str(field["source"][()]))
+        field = field.children[
+            0
+        ]  # only the first stream in the stream group can be edited currently
```

Comment:
Was this an existing limitation, or a new one? Is there a ticket to resolve it?

Reply:
I think it was an existing one, i'm not sure if there is a ticket to resolve it. i'm not really sure what the best way is of resolving it unless we add a `name` field to each stream so it can be searched as usual